README.md

```diff
@@ -130,45 +130,10 @@ Performance][kubecon-cortex-201]" ([video][kubecon-cortex-201-video], [slides][k
 [promcon-2016-video]: https://youtu.be/3Tb4Wc0kfCM
 [promcon-2016-slides]: http://www.slideshare.net/weaveworks/project-frankenstein-a-multitenant-horizontally-scalable-prometheus-as-a-service
 
-## <a name="help"></a>Getting Help
-
-If you have any questions about Cortex:
-
-- Ask a question on the [Cortex Slack channel](https://cloud-native.slack.com/messages/cortex/). To invite yourself to the CNCF Slack, visit http://slack.cncf.io/.
-- <a href="https://github.com/cortexproject/cortex/issues/new">File an issue.</a>
-- Send an email to <a href="mailto:cortex-users@lists.cncf.io">cortex-users@lists.cncf.io</a>
+## Hosted Cortex
 
-Your feedback is always welcome.
-
-For security issues see https://github.com/cortexproject/cortex/security/policy
-
-## Community Meetings
-
-The Cortex community call happens every two weeks on Thursday, alternating at 1200 UTC and 1700 UTC. Meeting notes are held [here](https://docs.google.com/document/d/1shtXSAqp3t7fiC-9uZcKkq3mgwsItAJlH6YW6x1joZo/edit).
-
-To see meeting calendar:
-
-- See the calendar [in your browser (time zone will be UTC)](https://calendar.google.com/calendar/u/0/embed?src=cncf-cortex-maintainers@lists.cncf.io&ctz=UTC).
-- If you use Google Calendar, [add the Cortex's calendar to your own Google Calendar](https://calendar.google.com/calendar/u/0?cid=Y25jZi1jb3J0ZXgtbWFpbnRhaW5lcnNAbGlzdHMuY25jZi5pbw).
-- You can also just [download the .ics file](https://calendar.google.com/calendar/ical/cncf-cortex-maintainers%40lists.cncf.io/public/basic.ics).
-
-## Hosted Cortex (Prometheus as a service)
-
-There are several commercial services where you can use Cortex
-on-demand:
+There are several commercial services where you can use Cortex on-demand:
 
 ### Amazon Managed Service for Prometheus (AMP)
 
-[Amazon Managed Service for Prometheus (AMP)](https://aws.amazon.com/prometheus/) is a Prometheus-compatible monitoring service that makes it easy to monitor containerized applications at scale. It is a highly available, secure, and managed monitoring for your containers. Get started [here](https://console.aws.amazon.com/prometheus/home). To learn more about the AMP, reference our [documentation](https://docs.aws.amazon.com/prometheus/latest/userguide/what-is-Amazon-Managed-Service-Prometheus.html) and [Getting Started with AMP blog](https://aws.amazon.com/blogs/mt/getting-started-amazon-managed-service-for-prometheus/).
-
-## Emeritus Maintainers
-
-* Peter Štibraný @pstibrany
-* Marco Pracucci @pracucci
-* Bryan Boreham @bboreham
-* Goutham Veeramachaneni @gouthamve
-* Jacob Lisi @jtlisi
-* Tom Wilkie @tomwilkie
```

Comment:
We can't remove this part, it's in the GOVERNANCE https://github.com/cortexproject/cortex/blob/master/GOVERNANCE.md?plain=1#L32-L33

Reply:
Agree - we should keep this section.